### PR TITLE
Change SDK project refs to nuget packages

### PR DESCRIPTION
**Pull request by EasyRhin0**
We need to update the Newtonsoft package ref in the SDK to v11 the next time we publish. Having the Viewer reference a newer version causes a very weird and inexplicable exception.

### CODE REVIEW

**Comment by EasyRhin0** [comment]
One note: this project uses the same strong-name key as the SDK. I'm not sure if that is desirable, acceptable, unacceptable, etc.

File: src/Sarif.Viewer.VisualStudio/packages.config [hunk 1]
```diff
@@ -8,6 +9,7 @@
   <package id="EnvDTE90a" version="9.0.2" targetFramework="net461" />
   <package id="Microsoft.ApplicationInsights" version="2.5.1" targetFramework="net461" />
   <package id="Microsoft.Build.Framework" version="15.6.82" targetFramework="net461" />
+  <package id="Microsoft.CodeAnalysis.BinSkim" version="1.4.5" targetFramework="net461" />
```
Comment (michaelcfanning) [inline src/Sarif.Viewer.VisualStudio/packages.config]: > <package id="Microsoft.CodeAnalysis.BinSkim" v [](start = 1, length = 47)

why is this required? didn't we shut down the binskim analysis?

Comment (EasyRhin0) [reply src/Sarif.Viewer.VisualStudio/packages.config]: It's a dependency of the Sarif.Sdk package. I'll file an issue for this and another nuget package fix we need.
